Update SPECT examples for new SIRF_data

samdporter opened the pull request:
Removes requirement for confusing `image = image.zoom_image(size=(12, -1, -1))`

SIRF_data SPECT template_sinogram updated so resolution matches PET and has only 1 slice

KrisThielemans commented:
> SIRF_data SPECT template_sinogram updated so resolution matches PET and has only 1 slice

where did you update the template?

samdporter commented:
https://github.com/SyneRBI/SIRF_data/pull/6

Added a PET template_sinogram.(h)s and changed
For SPECT changes matrix sizes, number of projections and scaling factor

samdporter commented:
Tested these. Work OK.

KrisThielemans commented:
thanks. I'm going to squash-merge this one if ok. quite a lot of merges that seem irrelevant.
